src/webgpu/api/operation/copyBetweenLinearDataAndTexture.spec.ts

```diff
@@ -946,9 +958,9 @@ g.test('copy_various_mip_levels')
           mipLevel: 6,
         },
       ])
-      .combine(poptions('format', kSizedTextureFormats))
-      .filter(formatCanBeTested)
-      .expand(textureSizeExpander)
+      .expand(({ mipLevel, _texturePhysicalSizeAtMipLevelInBlocks }) =>
+        textureSizeExpander({ mipLevel, _texturePhysicalSizeAtMipLevelInBlocks, format: p.format })
```

Comment:
Was this change needed because of the case/subcase change? Maybe textureSizeExpander could just be `computeTestTextureSizes` since it is no longer a functor.

Reply:
Done. Though one thing I learned from this patch I'd like if it could become a functor again but it would be complex (requires passing the cases() object into subcases() which means making .combine() return a new object instead of mutating a builder)